package.json

```diff
@@ -40,9 +40,11 @@
   },
   "scripts": {
     "build": "sl-scripts build && node ./scripts/prepare-cli",
+    "postbuild": "yarn copy.rulesets --verbose",
     "build.binary": "yarn build && yarn build.manifest && cp yarn.lock dist && cd dist && yarn && pkg . --out-path ./bin",
     "build.docs": "sl-scripts build:typedoc",
     "build.manifest": "cd dist && oclif-dev manifest",
+    "copy.rulesets": "copyfiles -u 1 \"src/**/ruleset.json\" dist",
```

Comment:
Let's ditch these commands if we arent doing the TS>YAML build thingy.

Reply:
I think this is still required, we need to copy the rules in the destination directory we're publishing for npm, right? @chris-miaskowski

Reply:
It's still required, TS does not copy JSON files for us, we need to do it manually.

Reply:
OOOOH because we put them in src, right.